rocAL/rocAL/include/api/rocal_api_data_transfer.h

```diff
@@ -24,8 +24,15 @@ THE SOFTWARE.
 #define MIVISIONX_ROCAL_API_DATA_TRANSFER_H
 #include "rocal_api_types.h"
 
+/*! \brief
+ *
+*/
 extern "C"  RocalStatus   ROCAL_API_CALL rocalCopyToOutput(RocalContext context, unsigned char * out_ptr, size_t out_size);
 
+extern "C"  RocalStatus   ROCAL_API_CALL rocalCopyToTensorOutput(RocalContext context, std::vector<void *> &out_ptr);
+/*! \brief
+ *
+*/
 extern "C"  RocalStatus   ROCAL_API_CALL rocalCopyToOutputTensor32(RocalContext rocal_context, float *out_ptr,
```

Comment:
Are we using this API or will we introduce in future?
If not remove it.

Reply:
Removed